vault/expiration.go

```diff
@@ -94,6 +98,7 @@ type ExpirationManager struct {
 	// TTL zero, which we want to count but have no timer associated.
 	pending     sync.Map
 	nonexpiring sync.Map
+	zombies     sync.Map
```

Comment:
A short description of what we expect this to contain might be useful here?

Reply:
894f249bab